Assets/Plugins/Source/Editor/EditorWindows/EOSSettingsWindow.cs

```diff
@@ -347,6 +349,12 @@ private void OnDefaultGUI()
             GUIEditorUtility.AssigningBoolField("Always send Input to Overlay",
                 ref mainEOSConfigFile.alwaysSendInputToOverlay, 190,
                 "If true, the plugin will always send input to the overlay from the C# side to native, and handle showing the overlay. This doesn't always mean input makes it to the EOS SDK.");
+
+            InputStateButtonFlags toggleFriendsButtonCombinationEnum = mainEOSConfigFile.GetToggleFriendsButtonCombinationFlags();
+            GUIEditorUtility.AssigningEnumField<InputStateButtonFlags>("Default Activate Overlay Button",
+                ref toggleFriendsButtonCombinationEnum, 190,
+                "Users can press the button(s) associated with this value to activate the Epic Social Overlay. Not all combinations are valid; the SDK will log an error at the start of runtime if an invalid combination is selected.");
+            mainEOSConfigFile.toggleFriendsButtonCombination = toggleFriendsButtonCombinationEnum.FlagsToStrings();
```

Comment:
This call to `FlagsToStrings()` can be replaced with the following call using [`EnumUtility.cs`](https://github.com/PlayEveryWare/eos_plugin_for_unity/blob/2a18ec7e3fddff1c35baf37195a9d7989be1c4bd/com.playeveryware.eos/Runtime/Core/Utility/EnumUtility.cs):
```c#
EnumUtility<InputStateButtonFlags>.GetEnumerator(toggleFriendsButtonCombinationEnum)
    .Select(enumValue => enumValue.ToString())
    .ToList();
```
You will also need to add a using reference at the top of the file:
```c#
using System.Linq;
```
(which is fine to do since this is all Editor code)

Reply:
Alertness raised at seeing LINQ, and then lowered on seeing that yup this is editor code. We'll do that here.

Reply:
That means we can get rid of FlagsToStrings, so I'll do so now. I was actually uncertain which way you'd lean, on whether there should be a function to do it or using this.

Reply:
Implemented above as suggested.